src/main/java/com/dawn/jat/illuminati/post/controller/PostExceptionController.java

```diff
@@ -0,0 +1,15 @@
+package com.dawn.jat.illuminati.post.controller;
+
+import com.dawn.jat.illuminati.post.PostNotfoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
```

Comment:
Should put it in src/main/java/com/dawn/jat/illuminati/post/advice

Reply:
Moved to advice folder